pkg/yang/identity.go

```diff
@@ -26,8 +26,10 @@ import (
 // to be identified by their module and name.
 type identityDictionary struct {
 	mu sync.Mutex
-	// TODO(wenovus): The key should not be prefix:ident-name, but
-	// module-name:ident-name. Prefixes are NOT unique.
+	// dict is a global cache of identities keyed by
+	// modulename:identityname, where modulename is the full name of the
+	// module to which the identity belongs. If the identity were defined
+	// in a submodule, then the belonging module name is used instead.
```

Comment:
(nit) `parent module name is used instead`.

Reply:
Done.